mobile/src/framework/notifications/notifications-base.package.ts

```diff
@@ -13,16 +13,20 @@ class BaseNotifications implements Notification {
     }
 
     public showError(settings: ShowMessageArguments): void {
+        const { title, text } = settings;
```

Comment:
Not a blocker, but you can create a private method and reuse it in the public methods.

It will be much better, than copy paste 
```
            text1: title,
            text2: text,
```